recipes-extended/xen/xen-source.inc

```diff
@@ -1,5 +1,13 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+
 LIC_FILES_CHKSUM = "file://COPYING;md5=d1a1e216f80b6d8da95fec897d0dbec9"
 
 XEN_URL = "git://github.com/xen-troops/xen.git;protocol=https;branch=xen-4.19-xt0.1"
 XEN_REL = "4.19"
 XEN_REV = "${AUTOREV}"
+
```

Comment:
This does not belong to `xen-source.inc`

Please add `xen.inc` and move this append there.

Reply:
Done